doc/release-tasks.md

```diff
@@ -41,6 +41,10 @@ GitHub.
 - [ ] Check that we don't call functions like
   `ReplicatedMesh::active_local_elements_end()` inside `for`-loop declarations
   since the end iterator is expensive to compute.
+- [ ] Check that we always get unique `HierarchyDataOps` by passing `true` as
```

Comment:
This was originally set up as a way to reduce memory usage...

If we always want unique `HierarchyDataOps` objects should we get rid of the option to share them?

Reply:
Oh, whoops, this is an internal SAMRAI thing, so we just need to make sure we cache these ourselves as needed to reduce memory usage.

Reply:
I don't see the point of caching these objects at all - the size of each one can be measured in bytes since all they do is call the correct functions (i.e., cell-centered, side-centered, or whatever) for the specified variable.

Either way, since its a SAMRAI function, we should just try to avoid it.